src/SFA.DAS.ApplyService.Web/appsettings.json

```diff
@@ -443,6 +475,15 @@
           "Action": "TaskList"
         }
       ]
+    },
+      "PageId": "10005",
```

Comment:
you're missing an opening { here I think....

Reply:
Good spot! Fixed